.evergreen/setup-mongodb-aws-auth-tests.sh

```diff
@@ -23,8 +23,6 @@ cd $DRIVERS_TOOLS/.evergreen/auth_aws
 
 cd $BEFORE
 
-npm install --no-save aws4
```

Comment:
The optional_require.test.ts file has been throwing an unhandled rejection as we test for importing aws4. So this is now tracked as a devDep like our other normal peers